app/components/EmptyState/index.tsx

```diff
@@ -4,20 +4,13 @@ import styles from './EmptyState.css';
 import type { ReactNode } from 'react';
 
 type Props = {
-  /** name of icon */
   icon?: string;
-
-  /** html to display in an EmptyState */
   children?: ReactNode;
   size?: number;
   className?: string;
 };
 
-/**
- * A basic EmptyState component
- *
- */
-const EmptyState = ({ icon, size = 88, className, children }: Props) => (
+const EmptyState = ({ icon, size = 45, className, children }: Props) => (
   <div className={cx(styles.container, icon && styles.centered, className)}>
```

Comment:
Couldn't a Flex component be used instead?

Reply:
Meh we only want it "flexed" when there's an icon, so I figured it was best to just keep the `icon && styles.centered`